Modules/Synchronization/test/Synchronization.Application.Tests/Tests/IntegrationEvents/IdentityDeletionProcessStatusChangedIntegrationEventHandlerTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Backbone.BuildingBlocks.Application.Abstractions.Infrastructure.EventBus;
+using Backbone.DevelopmentKit.Identity.ValueObjects;
+using Backbone.Modules.Synchronization.Application.Infrastructure;
+using Backbone.Modules.Synchronization.Application.IntegrationEvents.Incoming.IdentityDeletionProcessStatusChanged;
+using Backbone.Modules.Synchronization.Application.IntegrationEvents.Outgoing;
+using Backbone.Modules.Synchronization.Domain.Entities.Sync;
+using FakeItEasy;
+using Microsoft.Extensions.Logging;
+using Xunit;
+
+namespace Backbone.Modules.Synchronization.Application.Tests.Tests.IntegrationEvents;
+public class IdentityDeletionProcessStatusChangedIntegrationEventHandlerTests
+{
+    [Fact]
+    public async Task Creates_an_external_event()
```

Comment:
The test name promises to test that an external event is created. But what it actually does it is tests if an integration event is published.

You should actually do both (=> write 2 tests)